docs/rfcs/015-how_we_work/code_reviews.md

```diff
@@ -0,0 +1,23 @@
+# Code Reviews
+
+Code reviews are intended to ensure code changes meet a high standard and to share best practise between team members.
```

Comment:
what about sharing with the rest of the team (non-devs) what is going out by being transparent. Commenting shouldn't only be on code in the case of function / business logic, and thus PRs in this field should be clear in their description as to what they are doing if someone who doesn't read code will be looking at it.

Reply:
Can you either suggest an inline change or clarify in a future PR?

Reply:
Code reviews are intended to ensure code changes meet a high standard and to share best practise between team members.

```suggestion
Code reviews are intended to keep the quality of code high and understandable, while ensuring we are working in an iterative, pragmatic way.

They also help share best practise between team members.
```